### PR TITLE
Gyakorlat megoldás: TorpedoStore.java

### PR DESCRIPTION
**Pull request by akedves**
A SonarLint által jelölt problémák javítása a TorpedoStore.java fájlban.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 */
 public class TorpedoStore {
 
+  private Random generator = new Random();
```
Comment (alittahu) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:12]: Thats the way boi

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -30,18 +31,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (alittahu) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:34]: Yes, you need to use throw

Comment (alittahu) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:34]: Yes, you need to use throw

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
Comment (alittahu) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Yes, using Random as a single variable is a good way to use random

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 3]
```diff
@@ -30,18 +31,17 @@ public TorpedoStore(int numberOfTorpedos){
...
     boolean success = false;
 
     // simulate random overheating of the launcher bay which prevents firing
-    Random generator = new Random();
     double r = generator.nextDouble();
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (alittahu) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:44]: By =- i am sure u meant -=, so good job

**Review by alittahu — CHANGES_REQUESTED** [review]
Such a nice job, keep up the good work

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (akedves) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:34]: Ez pontosan így van

**Review by alittahu — APPROVED** [review]
Good to go

**Pull request merged**